config/pulse.php

```diff
@@ -164,6 +164,7 @@
 
         Recorders\Servers::class => [
             'server_name' => env('PULSE_SERVER_NAME', gethostname()),
+            'ignore_after' => env('PULSE_SERVER_IGNORE_AFTER'),
```

Comment:
I don't feel this should be a configuration for the recorder.

The recorder does not ignore the server after the given time, only the card does.

There isn't a 1:1 link between card and recorder.

I feel we should make this something that is configured at the card level.

`<livewire:pulse.servers :ignore-after="Interval::hours(6)" cols="full" />`

Reply:
Great point! Totally agree :+1:

Reply:
Yep, that's the right way to think about it, great point! I'll try and produce a (new?) pull request tomorrow.

Reply:
Please check the last commit.. Is this the way to go?
I'm not sure about the interval feature, what do you think?
`<livewire:pulse.servers :ignore-after="600" cols="full" />`
`<livewire:pulse.servers ignore-after="10 minutes" cols="full" />`